docs/source/api_examples_source/brainglobe_atlasapi.py

```diff
@@ -0,0 +1,198 @@
+"""
+BrainGlobe Atlas API
+====================
+Using the BrainGlobe Atlas API to fetch and inspect an atlas
+"""
+
+
+# %%
+# Import the Atlas API and find an atlas
+# --------------------------------------
+
+# %%
+# Import the API and some other tools
+
+from brainglobe_atlasapi import BrainGlobeAtlas
+from pprint import pprint # to format printed data nicely
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import cm
+
+# %%
+# To know what atlases are available through BrainGlobe, we can use the `show_atlases` function
+# (requires an internet connection):
+
+from brainglobe_atlasapi import show_atlases
+show_atlases()
+
+# %%
+# Creating a `BrainGlobeAtlas` object
+# -----------------------------------
+# To instantiate a `BrainGlobeAtlas` object, we need to instantiate it with the atlas name. The first time we use it, a
+# version of this atlas files will be downloaded from the `remote GIN repository <http://gin.g-node.org/brainglobe/atlases>`_
+# and stored on your local machine (by default, in `~/.brainglobe`):
+
+atlas = BrainGlobeAtlas("allen_mouse_100um", check_latest=False)
+
+# %%
+# .. admonition:: Atlas quality
+#   :class: note
+#
+#   Usually BrainGlobe only packages existing atlases. The quality of the underlying images and atlas ontology
+#   relies on the original atlas data, and this can vary considerably. In this example, we choose the "allen_mouse_100um"
+#   atlas, as it is relatively small, but high quality.
+
+
+
+
+# %%
+# Using the atlas
+# ---------------
+# A BrainGlobe atlas is a convenient API for interacting with an anatomical atlas. BrainGlobe atlases contain:
+#
+# * Metadata
+# * Reference anatomical stack
+# * Region annotation stack
+# * Hemisphere annotation stack
+# * Description of the region hierarchy
+# * Meshes for the regions
+
+# %%
+# Metadata
+# --------
+# All atlases have a standard set of metatata describing their source, species, resolution, etc:
+
+metadata = atlas.metadata
+pprint(metadata)
+
+# %%
+# Anatomical, annotation and hemispheres stacks
+# ---------------------------------------------
+
+# %%
+# Anatomical reference (or template) image:
+
+reference = atlas.reference
+
+# Get the middle section and plot
+middle_section = reference.shape[0] // 2
+
+plt.imshow(reference[middle_section,:,:], cmap='gray')
+
+# %%
+# Annotations stack
+annotation = atlas.annotation
+
+# Create a cyclic colormap due to the high values in the Allen atlas
+N = 512
+colors = cm.get_cmap('tab20', N)
```

Comment:
Removes deprecation warning.

```suggestion
colors = cm.get_cmap('tab20').resample(N)
```